modules/light-clients/08-wasm/types/mock_engine_test.go

```diff
@@ -13,27 +13,40 @@ import (
 
 const DefaultGasUsed = uint64(1)
 
-var _ ibcwasm.WasmEngine = (*MockWasmEngine)(nil)
+var (
+	_ ibcwasm.WasmEngine = (*MockWasmEngine)(nil)
 
-type QueryFn func(codeID wasmvm.Checksum, env wasmvmtypes.Env, queryMsg []byte, store wasmvm.KVStore, goapi wasmvm.GoAPI, querier wasmvm.Querier, gasMeter wasmvm.GasMeter, gasLimit uint64, deserCost wasmvmtypes.UFraction) ([]byte, uint64, error)
+	// queryTypes contains all the possible query message types.
+	queryTypes = [...]any{StatusMsg{}, ExportMetadataMsg{}, TimestampAtHeightMsg{}, VerifyClientMessageMsg{}, CheckForMisbehaviourMsg{}}
```

Comment:
Actually wondering now could the `any` here be a type constrain since it's a not a key in a map. This would only be a minor improvement but maybe a nice to have if it works.